src/components/CLI/CLI.tsx

```diff
@@ -1 +1,47 @@
-// Feature specific
\ No newline at end of file
+
+'use client';
+
+import React from 'react';
+import { useEffect, useRef } from 'react';
+import { Dice1 } from 'lucide-react';
+
+interface CliProps {
+  output: string[];
+  command: string;
+  setCommand: React.Dispatch<React.SetStateAction<string>>;
+  handleCommand: React.KeyboardEventHandler<HTMLInputElement>;
+}
+
+export default function Cli({ output, command, setCommand, handleCommand }: CliProps) {
+  const terminalRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (terminalRef.current) {
+      terminalRef.current.scrollTop = terminalRef.current.scrollHeight;
+    }
+  }, [output]);
+
+  return (
+    <div className="flex-grow flex flex-col">
+      <div ref={terminalRef} className="flex-grow overflow-auto mb-4 font-mono">
+        {output.map((line, index) => (
+          <div key={index} className={line.startsWith('dice>') ? 'text-blue-400' : 'text-green-400'}>
+            {line}
+          </div>
+        ))}
+      </div>
+      <div className="flex items-center bg-gray-700 rounded px-2 mb-4">
```

Comment:
@rishavvajpayee shall we try to maintain similarity with `redis-cli`? 
For reference:
<img width="826" alt="image" src="https://github.com/user-attachments/assets/d1527657-be9b-42f8-8dea-3bb0cd019298">